must-gather/common

```diff
@@ -17,7 +17,7 @@
 
 set -eu -o pipefail
 
-export KEPLER_NS="kepler-operator"
+export KEPLER_NS=$(oc get ns -l app.kubernetes.io/managed-by=kepler-operator --output=jsonpath='{.items[*].metadata.name}')
```

Comment:
perhaps only the first item instead of all items in the list? 
`jsonpath={.items[0].metadata.name}` ?

Reply:
ideally there should be only 1.

Reply:
yes, indeed .. already approved.
In a different PR, I think it is better to iterate on all namespaces to capture the details. That way we get all namespaces (perhaps user has both community and power-mon installed) etc ...